docs/configuration/reference/decorators.md

```diff
@@ -0,0 +1,91 @@
+# `decorators`
+
+## Introduction
+
+The `decorators` configuration section defines the transformation steps that are applied to your API description when it is bundled.
+Both the [built-in decorators](../../decorators.md) and [decorators from custom plugins](../../custom-plugins/custom-decorators.md) are configured this way.
+The `decorators` block can be used at the root of a configuration file, or inside an [API-specific section](./apis.md).
+
+## Options
+
+{% table %}
+
+- Option
+- Type
+- Description
+
+---
+
+- {decorator name}
+- string _or_ [Decorator object](#decorator-object)
+- Required. You can add as many decorators as you wish. The keys must be either built-in decorators (for example `info-description-override`), or a decorator from a plugin (for example `tags-plugin/no-unused-tags`). Set the value to `on` or `off` to enable or disable a decorator, or use a [Decorator object](#decorator-object) to configure additional options for a specific decorator.
+
+{% /table %}
+
+### Decorator object
+
+{% table %}
+
+- Option
+- Type
+- Description
+
+---
+
+- {additional properties}
+- any
+- Some decorators support additional configuration, check the documentation for each specific decorator to discover the values that can be used. For example the [`filter-out` decorator](../../decorators/filter-out.md) supports `property` and `value` settings for what to filter.
+
+---
+
+- severity
+- string
+- Severity level of any problems reported with this decorator. Setting this option is usually only useful for troubleshooting purposes. The value must be one of `error`, `warn`, or `off`, where `off` disables the decorator.
```

Comment:
Is there a default?

Reply:
I'm not sure, I didn't know this existed until I tried to write the doc and found it!